WordPress/src/main/java/org/wordpress/android/ui/accounts/SignInFragment.java

```diff
@@ -132,6 +132,7 @@
 
     private OnMagicLinkRequestInteraction mListener;
     private String mToken = "";
+    private boolean mInhibitMagicLogin;
```

Comment:
I'd prefer this boolean to be mIsMagicLogin. Negative booleans are confusing. This would also require flipping all logic, but I don't think that should be too hard.